### ISSUE TITLE
How to choose hyperparameters?

### ISSUE THREAD

**Issue by PlusRoss** [liqimai/Efficient-SSL#2]
Hi,

Thanks a lot for your excellent work. It seems that your paper focuses on the experiment setting without validation data. Then how do you choose the hyperparameters such as k, alpha, dropout, L2 regularization rate? Did you perform a grid search according to the lowest training loss in 200 steps?
